app/webpacker/controllers/talk-to-us_controller.js

```diff
@@ -0,0 +1,24 @@
+import { Controller } from "stimulus"
+
+export default class extends Controller {
+
+    static targets = ["chat", "launch", "tta"];
```

Comment:
You don't need the `launch` target, it's just the action being used

Reply:
nice one